src/java.desktop/share/classes/javax/swing/JLabel.java

```diff
@@ -874,8 +874,10 @@ public void setHorizontalTextPosition(int textPosition) {
 
 
     /**
-     * This is overridden to return false if the current Icon's Image is
-     * not equal to the passed in Image <code>img</code>.
+     * If the component is not showing or either the icon or disabled
+     * icon is not an <code>ImageIcon</code> with an <code>Image</code>
+     * equal to the passed in <code>Image</code>, return <code>false</code>;
+     * otherwise it will delegate to the super-class.
      *
```

Comment:
Hmm. I just noticed this doesn't have any @params .. we should add them since we are here.
I don't think they need to be included in the CSR since it isn't relevant.
Also let's use {@code ..} style since we are completely replacing all the text.